scripts/build-aci

```diff
@@ -1,11 +1,29 @@
-#!/usr/bin/bash
+#!/bin/bash
+
+# A non-installed actool can be used, for example:
+# ACTOOL=../../../appc/spec/bin/actool
+ACTOOL=${ACTOOL:-actool}
```

Comment:
I will add that test, and also check that $IMAGEDIR is an empty writeable directory.